aptos-move/aptos-debugger/src/aptos_debugger.rs

```diff
@@ -98,7 +98,8 @@ impl AptosDebugger {
             .map_err(|err| format_err!("Unexpected VM Error: {:?}", err))?;
 
         // TODO(Gas): revisit this.
-        let resolver = state_view.as_move_resolver();
+        let resolver: aptos_vm::data_cache::StorageAdapter<'_, DebuggerStateView> =
```

Comment:
Why this change?

Reply:
Hmm... Not sure why I changed this... Could be the legacy of some rebasing. Got it reverted.